pkg/controller/servicebinding/binder_test.go

```diff
@@ -375,6 +335,25 @@ func TestBinderApplicationName(t *testing.T) {
 	})
 }
 
+func TestBinderAppendEnvVar(t *testing.T) {
+	envName := "lastbound"
+	envList := []corev1.EnvVar{
+		corev1.EnvVar{
+			Name:  envName,
+			Value: "lastboundvalue",
+		},
+	}
+
+	b := &binder{}
+	updatedEnvVarList := b.appendEnvVar(envList, envName, "someothervalue")
+
+	// validate that no new key is added.
+	// the existing key should be overwritten with the new value.
+
+	require.Len(t, updatedEnvVarList, 1)
+	require.Equal(t, updatedEnvVarList[0].Value, "someothervalue")
+}
+
```

Comment:
It seems this method has been relocated but hasn't changed; can you revert this?

Reply:
Done dce1a2b